src/test/java/io/in_toto/models/LinkTest.java

```diff
@@ -0,0 +1,146 @@
+package io.in_toto.models;
+
+import io.in_toto.models.Artifact.ArtifactHash;
+import io.in_toto.models.Link;
+import io.in_toto.keys.RSAKey;
+import io.in_toto.keys.Key;
+
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.HashMap;
+
+import java.io.File;
+import java.io.IOException;
+
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.migrationsupport.rules.EnableRuleMigrationSupport;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.ExpectedException;
+import org.junit.Rule;
+
+/**
+ * Link-specific tests
+ */
+ 
+@DisplayName("Link-specific tests")
+@EnableRuleMigrationSupport
+@TestInstance(Lifecycle.PER_CLASS)
+class LinkTest
+{
```

Comment:
Identation in this class needs to be reviewed.

Reply:
done

Reply:
Just realized this is still using tabs...

Reply:
Is it ok now? 95d2a8bd39acec99860798b192cf98455bf9aeff

Reply:
all the files were corrected with sed 55bc067afccc21e22722acae624627380affb73f